pkg/fixtures/purger_blob.go

```diff
@@ -0,0 +1,45 @@
+package fixtures
+
+import (
+	"context"
+	"github.com/applike/gosoline/pkg/blob"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+)
+
+type blobPurger struct {
+	logger      mon.Logger
+	batchRunner blob.BatchRunner
+	store       blob.Store
+	bucketName  string
```

Comment:
i just realized, that you are not even using this var. In that case, please remove it completely.

Reply:
True. Done.